### ISSUE TITLE
WPT test for Blob constructor has unexpected behaviour

### ISSUE THREAD

**Issue by izgzhen** [servo/servo#11075]
In `tests/wpt/metadata/FileAPI/blob/Blob-constructor.html.ini`, the second test:

     [Passing an platform object that supports indexed properties as the blobParts array should work (window).]
    expected: FAIL

you can delete this line, modify it to `PASS` or whatever else, the test will still pass.

Look at the `tests/wpt/web-platform-tests/FileAPI/blob/Blob-constructor.html` and we found:

```js
test_blob(function() {
  var select = document.createElement("select");
  select.appendChild(document.createElement("option"));
  return new Blob(select);
}, {
  expected: "[object HTMLOptionElement]",
  type: "",
  desc: "Passing an platform object that supports indexed properties as the blobParts array should work (select)."
});
```

First, it is `select` but not `window`, but even if I changed the `select` to `window`, the test is still not working.

**Comment by izgzhen** [comment]
Ok ... I found another one testing `select` ... I will fix the test script later

**Comment by izgzhen** [comment]
@Ms2ger  ` [Passing an platform object that supports indexed properties as the blobParts array should work (window with custom toString).]` and `[Passing an platform object that supports indexed properties as the blobParts array should work (window).]` seem not existing in the `Blob-constructor.html` anymore. Should I delete the corresponding testing entry?

(BTW, looks like the `./mach test` doesn't complain even if some test in `ini` doesn't exist)

**Comment by jdm** [comment]
Yes, INI entries that don't correspond to actual tests are ignored by the test harness. It can lead to confusion, but I'm not sure if it's a big deal.

**Comment by izgzhen** [comment]
@jdm Ok, if "nothing happens" == "everything is good" then then it looks sensible to ignore it. I will fix this manually later.

**Comment by timvandermeij** [comment]
Fixed by https://github.com/servo/servo/commit/b93409bab0fa05a3f86e32e5c54ca874d3e5c732 ?

**Comment by ConnorGBrewster** [comment]
Fixed by #11078
